src/backend/distributed/commands/policy.c

```diff
@@ -29,7 +29,7 @@
 
 
 static const char * unparse_policy_command(const char aclchar);
-static void AddRangeTableEntryToQueryCompat(ParseState *parseState, Relation relation);
+void AddRangeTableEntryToQueryCompat(ParseState *parseState, Relation relation);
```

Comment:
I guess you should extern this function. I wonder if there is a more meaningful place for this function to be defined rather than `policy.c` because it becomes kind of meaningless to extern it in `/* policy.c -  forward declarations */`
https://github.com/citusdata/citus/blob/main/src/include/distributed/commands.h#L392